sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2837,6 +2837,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val LEGACY_ALLOW_READ_STREAMING_TEMP_VIEW =
+    buildConf("spark.sql.legacy.allowReadStreamingTempView")
```

Comment:
`allowReadingStreamingTempViewInBatchAPI`?

Reply:
Thanks, done in 2131507